test/declaration_test.ts

```diff
@@ -102,23 +102,27 @@ describe('classes', () => {
       expectTranslate('class X { constructor() { } }').to.equal(' class X { X ( ) { } }');
     });
     it('supports parameter properties', () => {
+      expectTranslate('class X { c: number; \n' +
```

Comment:
FYI these changes are unrelated, I just wanted to fix the lines >> 100 chars here.